widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetView.kt

```diff
@@ -8,29 +8,66 @@ import com.glia.widgets.di.Dependencies
 import com.glia.widgets.entrywidget.adapter.EntryWidgetAdapter
 import com.glia.widgets.entrywidget.adapter.EntryWidgetItemDecoration
 import com.glia.widgets.helper.getDrawableCompat
+import com.glia.widgets.view.unifiedui.applyLayerTheme
+import com.glia.widgets.view.unifiedui.theme.base.ButtonTheme
+import com.glia.widgets.view.unifiedui.theme.base.LayerTheme
+import com.glia.widgets.view.unifiedui.theme.base.TextTheme
+import com.glia.widgets.view.unifiedui.theme.entrywidget.EntryWidgetTheme
+import com.glia.widgets.view.unifiedui.theme.entrywidget.MediaTypeItemsTheme
 
 /**
  * EntryWidgetView provides a way to display the entry points for the user to start a chat, audio call, video call, or secure messaging.
  */
 internal class EntryWidgetView(
     context: Context,
-    viewType: EntryWidgetContract.ViewType
+    viewType: EntryWidgetContract.ViewType,
+    backgroundTheme: LayerTheme? = null,
+    mediaTypeItemsTheme: MediaTypeItemsTheme? = null,
+    errorTitleTheme: TextTheme? = null,
+    errorMessageTheme: TextTheme? = null,
+    errorButtonTheme: ButtonTheme? = null
 ) : RecyclerView(context, null, 0), EntryWidgetContract.View {
+
+    constructor(
+        context: Context,
+        viewType: EntryWidgetContract.ViewType,
+        entryWidgetTheme: EntryWidgetTheme?
+    ) : this(
+        context,
+        viewType,
+        entryWidgetTheme?.background,
+        entryWidgetTheme?.mediaTypeItems,
+        entryWidgetTheme?.errorTitle,
+        entryWidgetTheme?.errorMessage,
+        entryWidgetTheme?.errorButton
+    )
+
     var onDismissListener: (() -> Unit)? = null
 
     private lateinit var controller: EntryWidgetContract.Controller
-    private val adapter = EntryWidgetAdapter(viewType).also { setAdapter(it) }
+    private val adapter = EntryWidgetAdapter(
+        viewType, mediaTypeItemsTheme, errorTitleTheme, errorMessageTheme, errorButtonTheme
+    ).also { setAdapter(it) }
 
     init {
-        layoutManager = LinearLayoutManager(context)
+        isNestedScrollingEnabled = false
+        layoutManager = object : LinearLayoutManager(context) {
+            override fun canScrollVertically() = false
+        }
```

Comment:
It would be nice to have a comment on why this scrolling setting is needed here. Or instead, the comment could be extracted in a separate method with self explanatory name. For example _(I really have no idea what it is for)_: `upplyFixForScrollingOnSmallScreen()`

Reply:
Since the `Entry Widget` might be embedded in scrolling views, disabling the scroll for itself would prevent unintended scrolling behavior in such cases.

I will move this functionality to a separate extension function with a more descriptive name.